<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>整体信息</string>
     </attribute>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>271</width>
        <height>95</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>班级编号</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="t_cla_id"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="t_cla_name"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="t_cla_monitor"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>班  级  名</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>班长学号</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>60</y>
        <width>271</width>
        <height>64</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>学院编号</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="t_depa_id"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>学  院  名</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="t_depa_name"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>230</y>
        <width>581</width>
        <height>221</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="3">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>学号</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLineEdit" name="t_sex"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>姓       名</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>41</width>
           <height>25</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="t_nactive_place"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="t_class"/>
       </item>
       <item row="0" column="4">
        <widget class="QLineEdit" name="t_id"/>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="t_name"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="t_department"/>
       </item>
       <item row="2" column="4">
        <widget class="QLineEdit" name="t_birthday"/>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>性别</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>生日</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>班级编号</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>学院编号</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>籍       贯</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QDialogButtonBox" name="cls_ar">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>140</y>
        <width>152</width>
        <height>32</height>
       </rect>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
     <widget class="QDialogButtonBox" name="depa_ar">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>140</y>
        <width>152</width>
        <height>32</height>
       </rect>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
     <widget class="QDialogButtonBox" name="per_ar">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>470</y>
        <width>152</width>
        <height>32</height>
       </rect>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>0</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>班级信息录入</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>20</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>学院信息录入</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>180</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>个人信息录入</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>学籍变更情况</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>120</y>
        <width>581</width>
        <height>221</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="4">
        <widget class="QLineEdit" name="t_ch_stuid"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>变更代码</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>记  录  号</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="t_ch_id"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="t_ch_des"/>
       </item>
       <item row="1" column="4">
        <widget class="QLineEdit" name="t_ch_tm"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="t_chid"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>描       述</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>学       号</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>41</width>
           <height>25</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>记录时间</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QDialogButtonBox" name="chg">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>350</y>
        <width>152</width>
        <height>32</height>
       </rect>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_32">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>60</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>学籍变更信息录入</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>奖励情况</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>130</y>
        <width>581</width>
        <height>221</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="4">
        <widget class="QLineEdit" name="t_rw_stuid"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>变更代码</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>记  录  号</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="t_rw_id"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="t_rw_des"/>
       </item>
       <item row="1" column="4">
        <widget class="QLineEdit" name="t_rw_tm"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="t_rw_lv"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>描       述</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>学       号</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>41</width>
           <height>25</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>记录时间</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QDialogButtonBox" name="rw">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>360</y>
        <width>152</width>
        <height>32</height>
       </rect>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>70</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>奖励情况</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>处罚情况</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>110</y>
        <width>581</width>
        <height>221</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="4">
        <widget class="QLineEdit" name="t_pns_stuid"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>变更代码</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>记  录  号</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="t_pns_id"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="t_pns_ea"/>
       </item>
       <item row="1" column="4">
        <widget class="QLineEdit" name="t_pns_tm"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="t_pns_lv"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>是否生效</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>学      号</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>41</width>
           <height>25</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_27">
         <property name="text">
          <string>记录时间</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QLineEdit" name="t_pns_des"/>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="label_28">
         <property name="text">
          <string>描      述</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QDialogButtonBox" name="pns">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>350</y>
        <width>152</width>
        <height>32</height>
       </rect>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_34">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>60</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>处罚情况录入</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>学生情况查询和修改</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>701</width>
        <height>401</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>学号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>姓名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>性别</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>班级编号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>院系编号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>生日</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>籍贯</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="stu_query_Button">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>个人信息查询</string>
      </property>
     </widget>
     <widget class="QPushButton" name="chg_query_p">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>学籍变更查询</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rw_query_p">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>奖励记录查询</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pns__query_p">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>处罚记录查询</string>
      </property>
     </widget>
     <widget class="QPushButton" name="depa__query_p">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>10</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>院系信息查询</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cls__query_p">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>10</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>班级信息查询</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stu_clapushButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>470</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
     <widget class="QPushButton" name="del_p">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>470</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
     <widget class="QPushButton" name="update_ctl">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>470</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>更新</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
